packages/strapi-plugin-i18n/config/functions/migrations/content-type/index.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const { getService } = require('../../../../utils');
+const { DEFAULT_LOCALE } = require('../../../../constants');
+
+const getDefaultLocaleRows = (model, ORM) => {
+  if (model.orm === 'bookshelf') {
+    return ORM.knex('core_store')
+      .select('value')
+      .where({ key: 'plugin_i18n_default_locale' });
+  }
+
+  if (model.orm === 'mongoose') {
+    return strapi.models['core_store'].find({ key: 'plugin_i18n_default_locale' });
+  }
+};
+
+const updateLocale = (model, ORM, locale) => {
+  if (model.orm === 'bookshelf') {
+    return ORM.knex(model.collectionName)
+      .update({ locale })
+      .where({ locale: null });
+  }
+
+  if (model.orm === 'mongoose') {
+    return model.updateMany(
+      { $or: [{ locale: { $exists: false } }, { locale: null }] },
```

Comment:
Is there a reason not to overwrite ?

Reply:
To avoid destroying data if for any reasons there are some, just to be cautious